modules/cards/card_4_By_Ashish.html

```diff
@@ -0,0 +1,81 @@
+<style>
+    *{
+    margin: 0px;
+    padding: 0px;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+body{
```

Comment:
This would affect the entire project. do not style the entire body as your module may just be a section add to any website.

Reply:
You can use your parent/first "div" with class="container"

Reply:
hello hope you are doing great.

I have made few changes

Avoided directly styling the general tags(body, head, html etc) by adding class to them.

Sorry for the mistakes as this is my first time contributing.

Tell me if any other changes required.

Thank you.